README.md

```diff
@@ -249,6 +250,58 @@ Medium.
 - Idea discussion
 - [rustc_codegen_gcc](https://rust-lang.zulipchat.com/#narrow/channel/386786-rustc-codegen-gcc/)
 
+### Layout-handling-for-the-automatic-differentiation-feature
+
+**Description**
+
+Over the last year, support for automatic differentiation ('autodiff') was added to the Rust compiler. The autodiff tool which we are using (Enzyme) operates 
```

Comment:
I think that Enzyme is not very much known yet, it would be nice to add some link here, e.g. to https://enzyme.mit.edu/.